console/src/main/scripts/plugins/metrics/ts/appServerDetails.ts

```diff
@@ -685,14 +706,14 @@ module HawkularMetrics {
           .withName('CPU usage')
           .withDescription(description)
           .withActions(this.defaultAction)
-          .withTags({ resourceId: this.feedId, resourceName: cpuResource['name'] })
+          .withTags({resourceId: this.feedId, resourceName: cpuResource['name']})
           .withContext(
-          new AlertDefinitionContextBuilder(AlertDefinitionContext.TRESHOLD_TRIGGER_TYPE)
```

Comment:
Please make the builders static. We shouldn't have to **new** an instance every time, and it just creates extra instances for no benefit (introduces unnecessary GC).